docs/examples.rst

```diff
@@ -0,0 +1,16 @@
+Examples
+========
+
+Here we catalog all the example Jupyter notebooks that have been created for
+the various functionality contained in `xrtpy`.
```

Comment:
This is the page that we'll eventually use to show a gallery of example Jupyter notebooks.